bin/parse_phenotype.py

```diff
@@ -47,8 +65,8 @@
             }
 
 # Curate non-session related data and parse into separate .tsv & .json files
-for name, cat in zip(["demographics", "height_weight", "diagnosis", "admin"], 
-    [demographics, height_weight, diagnosis, admin]): 
+for name, cat in zip(["demographics", "height_weight", "diagnosis", "admin", "BL_assessment", "F_assessment", "one_ses-baseline", "one_ses-oneweek"], 
```

Comment:
These new variables are clearly session related. I don't understand what you are trying to achieve.

Reply:
I understand that these are session related, but their names are not identical across assessment or across baseline & follow up. The way the script has been written dicates that assessments should all contain an assessment identifier (such as STAI or BPQ), this is not so in these assessments for whatever reason. Additionally there differences in naming the same thing across sessions. This is why I have done it in this way.

Reply:
You have found why `"demographics", "height_weight", "diagnosis", "admin"` are separated from things with readable assessment identifier. These are categories I defined posthoc. Now this is clearer for me - you identify some variables that probably belong to some cohesive category that was not coded in the source data. Next time explain that in comments in the code.